routes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+  function call($controller, $action) {
+    require_once('controllers/' . $controller . '_controller.php');
+
+    switch($controller) {
+      case 'movies':
+        $controller = new MoviesController();
+      break;
+      // case 'newController':
```

Comment:
You should just remove commented code. It doesn't run after all...

Reply:
oks!